core/arch/arm/plat-corstone1000/platform_config.h

```diff
@@ -0,0 +1,46 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2022, Arm Limited
+ */
+
+#ifndef PLATFORM_CONFIG_H
+#define PLATFORM_CONFIG_H
+
+#include <mm/generic_ram_layout.h>
```

Comment:
It looks like these two included aren't needed.

Reply:
Yes, @etienne-lms may be able to explain better but I think including `<mm/generic_ram_layout.h>` allows platforms to make sure that `TEE_RAM_VA_START` and `TEE_RAM_VA_SIZE` (and possibly others) are defined based on CFG values or default values. But since some platforms (used to?) hard-code values in their `platform_config.h`, the callers always include `<platform.h>` not `<mm/generic_ram_layout.h>`. There is one exception which does not look correct though: `core/arch/arm/plat-imx/drivers/tzc380.c` (but it probably doesn't matter because all i.MX flavors use the generic mechanism).